Compilation warnings

Recent changes have caused StockView to start giving some compile-time warnings for unused variables:

```
C:\ApsimX\ApsimNG\Views\StockView.cs(1024,17,1024,20): warning CS0168: The variable 'Idx' is declared but never used
C:\ApsimX\ApsimNG\Views\StockView.cs(1167,17,1167,26): warning CS0168: The variable 'iGenotype' is declared but never used
C:\ApsimX\ApsimNG\Views\StockView.cs(1168,18,1168,29): warning CS0219: The variable 'bIsPregnant' is assigned but its value is never used
C:\ApsimX\ApsimNG\Views\StockView.cs(1169,18,1169,30): warning CS0219: The variable 'bIsLactating' is assigned but its value is never used
C:\ApsimX\ApsimNG\Views\StockView.cs(1170,18,1170,27): warning CS0219: The variable 'bHasYoung' is assigned but its value is never used
```

--- a/ApsimNG/Views/StockView.cs
+++ b/ApsimNG/Views/StockView.cs
@@ -1021,7 +1021,6 @@ namespace UserInterface.Views
             int iGenotype;
             AnimalInits animalGroup;
             GrazType.AnimalType animalType;
-            int Idx;
 
             if (currentGroup < 0)
                 gbxAnimals.Hide();
@@ -1164,11 +1163,6 @@ namespace UserInterface.Views
         /// </summary>
         private void ParseCurrGroup()
         {
-            int iGenotype;
-            bool bIsPregnant = false;
-            bool bIsLactating = false;
-            bool bHasYoung = false;
-
             if (currentGroup >= 0)
             {
                 AnimalInits animalGroup = this.animalInits[currentGroup];

